src/XR/features/WebXRControllerPointerSelection.ts

```diff
@@ -366,9 +408,42 @@ export class WebXRControllerPointerSelection extends WebXRAbstractFeature {
                     scene.pointerY = this._screenCoordinatesRef.y;
                 }
             }
-            controllerData.pick = this._scene.pickWithRay(controllerData.tmpRay, this._scene.pointerMovePredicate || this.raySelectionPredicate);
 
-            const pick = controllerData.pick;
+            let pick = null;
+            if(controllerData.hoverMesh)
+            {
+                let hoverInfo = this._pickWithMesh(controllerData.hoverMesh, false);
+                let nearHover = hoverInfo && hoverInfo.pickedPoint && hoverInfo.hit;
+                if(nearHover)
+                {
+                    console.log("HOVER SUCCESS");
+                    controllerData.hover = hoverInfo;

```

Comment:
currently hover pick information is being populated in a different property "controllerData.hover". this is different from "controllerData.pick" that is used by pointermove events. Should i update hover to even use "controllerData.pick"  instead?